### PR TITLE
lavrivskyi-jv-oop-lesson-0

### PR DESCRIPTION
**Pull request by lavrivskyi**
implemented "login" method and "findByEmail" method

### CODE REVIEW

File: src/main/java/mate/academy/service/UserService.java [hunk 1]
```diff
@@ -14,7 +14,12 @@
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
+    public static User findByEmail(String email, String password) {
```
[the remark this answers is not in the document]
Comment (lavrivskyi) [reply src/main/java/mate/academy/service/UserService.java]: "findByEmail" method signature fixed

File: src/main/java/mate/academy/service/UserService.java [hunk 2]
```diff
@@ -14,7 +14,12 @@
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
+    public static User findByEmail(String email) {
```
Comment (Iryna-U) [inline src/main/java/mate/academy/service/UserService.java:17]: Please, don't change the method signature
```suggestion
    public User findByEmail(String email) {
```

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 3]
```diff
@@ -11,6 +13,11 @@
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User userData = UserService.findByEmail(email);
+        boolean passExists = false;
+        if (userData != null) {
+            passExists = UserService.findByEmail(email).getPassword().equals(password);
+        }
+        return passExists;
```
Comment (Iryna-U) [inline src/main/java/mate/academy/service/AuthenticationService.java:21]:
```suggestion
        User user = new UserService.findByEmail(email);
        if (user != null) {
            return user.getPassword().equals(password);
        }
        return false;
```

**Review by Iryna-U — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/service/UserService.java (hunk 2 above)
Comment (lavrivskyi) [reply src/main/java/mate/academy/service/UserService.java]: Fixed.

File: src/main/java/mate/academy/service/AuthenticationService.java (hunk 3 above)
Comment (lavrivskyi) [reply src/main/java/mate/academy/service/AuthenticationService.java]: Fixed.

**Review by Iryna-U — APPROVED** [review]

**Pull request closed without merging**